README.md

```diff
@@ -201,12 +203,12 @@ It is possible to disable / enable the spell checker by adding comments to your
 -   `/* spell-checker: enable */`
 -   `/* spellchecker: enable */`
 
-#### Example
+#### JavaScript Example
 
 ```javascript
 // cSpell:disable
 const wackyWord = ['zaallano', 'wooorrdd', 'zzooommmmmmmm'];
-/* cSpell:enable */
+// cSpell:enable //
```

Comment:
No need to change this one. There are two types of comments in JavaScript: line comments (`//`) and block comments (`/* ... */`).

It should be either: `/* cSpell:enable */` or `// cSpell:enable`, but not `// cSpell:enable //`